client/src/components/Header/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import logo from '../../assets/sneaker24-logo.png'
+
+export default function Header({ text }) {
```

Comment:
If you don't use the prop `text`, why do you have it? :)

Reply:
That's a good question. Don't know how this prop sneaked in 😂